addons/web/tooling/_jsconfig.json

```
@@ -25,29 +25,67 @@
             "@mail/*": ["addons/mail/static/src/*"],
             "@im_livechat/*": ["addons/im_livechat/static/src/*"],
             "@whatsapp/*": ["addons/whatsapp/static/src/*"],
+
             "@point_of_sale/*": ["addons/point_of_sale/static/src/*"],
+            "@pos_restaurant_stripe/*": ["pos_restaurant_stripe/static/src/*"],
+            "@pos_hr_restaurant/*": ["pos_hr_restaurant/static/src/*"],
+            "@pos_self_order/*": ["pos_self_order/static/src/*"],
```

```suggestion
            "@pos_self_order/*": ["addons/pos_self_order/static/src/*"],
```

You forgot `addons` that's why it wasn't working after refresh.

---

i'll change it now